Topics/Software_Engineering/Retries.md

```diff
@@ -0,0 +1,86 @@
+# Retries in web services
+
+Especially relevant for webserver applications, but useful for others, retries are really tricky to get right. 
+Retries and throttling are both terms used to talk about the _flow_ of traffic into a service. Often the 
+operators/developers of that service want to make guarantees about the rate of that flow, or otherwise direct 
+traffic.
```

Comment:
Though the term "retry" may seem intuitive, maybe adding a sentence to explicitly define it will help the reader. E.g., "retries are essentially sending the same request again"?